### ISSUE TITLE
Camera rendering issue in headless mode

### ISSUE THREAD

**Issue by YoruCathy** [empriselab/RCareWorld#180]
TL;DR: you still need a monitor.

The parameter `no-graphics` controls the rendering. If specified, it does true headless rendering and only computes the physics part. This enables it to run on GitHub CI but does not allow the camera to read anything. In this mode, the picture from the Python API is corrupted.

It runs in a ‘semi-headless' if remove `no-graphics.` There will be an empty black window without any rendering, but the system still needs to have some rendering capability. In this mode, the users will be able to get the pictures using the Python API. 

You can give this a try if you want to use a headless server. Remove the `no-graphics` param as shown [here](https://github.com/empriselab/RCareWorld/commit/5946aeb7951590e9b9f515ff6d47b3df35f1473c). And then you will need the following commands to configure your Ubuntu server
`sudo apt-get install xserver-xorg libglu1-mesa-dev freeglut3-dev mesa-common-dev libxmu-dev libxi-dev`
Then, restart your server and connect your server to a screen. The screen won't render anything, but there must be a display device connecting to your server.

```
# Configure nvidia-x
sudo nvidia-xconfig -a --use-display-device=None --virtual=1280x1024

# Configure environment variable
export DISPLAY=:1

# Launch a virtual display
sudo /usr/bin/X :1 &
```

I'm still keeping the `no-graphics` option in the code because we need the CI tests.

**Comment by YoruCathy** [comment]
https://github.com/empriselab/RCareWorld/pull/179 As shown in this PR, it's not able to open the Unity executables and run the test without rendering.
